catalog.html

```diff
@@ -319,22 +325,27 @@ <h2>ПАРА СЛОВ О ПЕРФОРАТОРАХ</h2>
 
     <div class="footer-social">
       <ul>
-        <li><a class=”social-button” href="#">
-            Vk
-          </a></li>
-        <li><a class=”social-button” href="#">
-            Facebook
-          </a></li>
-        <li><a class=”social-button” href="#">
-            Instagram
-          </a></li>
+        <li><a class=”social-button” href="#">Vk</a></li>
```

Comment:
```suggestion
        <li><a class="social-button" href="#">Vk</a></li>
```
допускаются только прямые кавычки. Заменить надо все вхождения.